src/aliceVision/image/io.cpp

```diff
@@ -26,15 +26,114 @@
 
 namespace fs = boost::filesystem;
 
+namespace
+{
+    std::string getDefaultColorConfigFilePath()
+    {
+        std::string configOCIOFilePath = "";
+
+        char const* ALICEVISION_OCIO = std::getenv("ALICEVISION_OCIO");
+        if (ALICEVISION_OCIO != NULL)
+        {
+            configOCIOFilePath = std::string(ALICEVISION_OCIO);
+            if (fs::exists(configOCIOFilePath))
+            {
+                // Check if a sRGB linear color space named "scene-linear Rec.709-sRGB" is present and set as scene_linear role 
+                oiio::ColorConfig colorConfig(configOCIOFilePath);
+                const std::string linearColorSpace = colorConfig.getColorSpaceNameByRole("scene_linear");
+                if (linearColorSpace == "scene-linear Rec.709-sRGB")
+                {
+                    ALICEVISION_LOG_TRACE("ALICEVISION_OCIO configuration file: '" << configOCIOFilePath << "' found.");
+                    return configOCIOFilePath;
+                }
+                else
+                {
+                    ALICEVISION_LOG_WARNING("ALICEVISION_OCIO configuration file is not valid: '" << configOCIOFilePath << "'.\n"
+                                            "The scene_linear role named \"scene-linear Rec.709-sRGB\" is required.\n"
+                                            "Skip this OCIO configuration file and use the embedded one.");
+                }
+            }
+            else if (configOCIOFilePath.empty())
+            {
+                ALICEVISION_LOG_TRACE("ALICEVISION_OCIO is empty.");
+            }
+            else
+            {
+                ALICEVISION_LOG_WARNING("ALICEVISION_OCIO is defined but does not point to an existing file: '" << configOCIOFilePath << "'");
+            }
+        }
+
+        // To be uncommented to take OCIO env var in consideration before using the enbedded config file
```

Comment:
Commented out code will bitrot. I think it would make sense to put everything in an `if` with condition hardcoded to false.

Reply:
ok